examples/animated-say/main.go

```diff
@@ -0,0 +1,56 @@
+// Package main illustrates how to make a method call to a remote
+// object. It uses the specialized proxy of the text to speech
+// service.
+package main
+
+import (
+	"flag"
+	"fmt"
+	"math/rand"
+
+	"github.com/lugu/qiloop/app"
+	"github.com/lugu/qiloop/bus/services"
+)
+
+func main() {
+	flag.Parse()
+	// session represents a connection to the service directory.
+	session, err := app.SessionFromFlag()
+	if err != nil {
+		panic(err)
+	}
+	defer session.Terminate()
+
+	// Access the specialized proxy constructor.
+	proxies := services.Services(session)
+
+	// Obtain a proxy to the service
+	ans, err := proxies.ALAnimatedSpeech(nil)
+	if err != nil {
+		panic(err)
+	}
+
+	err = ans.SetBodyLanguageEnabled(true)
+	if err != nil {
+		panic(err)
+	}
+	err = ans.SetBodyTalkEnabled(true)
+	if err != nil {
+		panic(err)
+	}
+
+	anime := getRandomExplain()
```

Comment:
maybe we can make it a little more portable using tags (pepper and nao may not have the same content).
i don't have a robot to test (!), but maybe something like this would work too:

```
animatedSentence := "^startTag(explain) I have always been a very good explainer, did you know? ^waitTag(explain)"
ans.Say(animatedSentence)
```

Reply:
Yes it does work - tested it on a v5 nao
You mentioned that you do not have a robot to test - but you work for Aldebaran right?
What is your motivation in building qiloop - are you assigned to this from your job or you do it on your own accord?

Reply:
i am not sure about my motivation, i started because i found it fun to reverse engineer qimessaging... then qiloop became my pet project during the commute and the weekend.
i don't develop it at work. also, hearing from your story i realized no one it taking care of those NAOv5 robots... so i guess i am on a mission now :)